internal/service/pushbasedlogexport/resource_test.go

```diff
@@ -0,0 +1,311 @@
+package pushbasedlogexport_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+const (
+	resourceName       = "mongodbatlas_push_based_log_export.test"
+	datasourceName     = "data.mongodbatlas_push_based_log_export.test"
+	nonEmptyPrefixPath = "push-log-prefix"
+	defaultPrefixPath  = ""
+)
+
+func TestAccPushBasedLogExport_basic(t *testing.T) {
```

Comment:
similar to above about using "basicTestCase" pattern with mig.CreateAndRunTest

Reply:
updated